schemas/common/autogeneratedResources.json

```diff
@@ -3217,6 +3217,9 @@
         {
           "$ref": "https://schema.management.azure.com/schemas/2020-08-01/Microsoft.ImportExport.json#/resourceDefinitions/jobs"
         },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2018-06-17-preview/microsoft.insights.json#/resourceDefinitions/workbooks"
```

Comment:
Please remove this - it's already referenced in 2019-04-01/deploymentTemplate.json